internal/cli/cmd/connectivity.go

```diff
@@ -157,6 +157,7 @@ func newCmdConnectivityTest(hooks Hooks) *cobra.Command {
 	cmd.Flags().MarkHidden("include-unsafe-tests")
 	cmd.Flags().BoolVar(&params.K8sLocalHostTest, "k8s-localhost-test", false, "Include tests which test for policy enforcement for the k8s entity on its own host")
 	cmd.Flags().MarkHidden("k8s-localhost-test")
+	cmd.Flags().BoolVar(&params.CiliumEndpointDisabled, "skip-cilium-endpoint-check", false, "Skip Cilium endpoint CRD check")
```

Comment:
Rather than add a flag, can we read the value of `disableEndpointCRD: "true"` from Cilium's live config.

This would be more accurate and easier to use.

Reply:
Done, thanks!